storage/client/queue/src/main/java/com/azure/storage/queue/Utility.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.storage.queue;
+
+import com.azure.core.implementation.util.ImplUtils;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.Locale;
+import java.util.TreeMap;
+
+
+public final class Utility {
```

Comment:
This class feels like it should be in common package, not queue.

Reply:
Alan has another PR to extract the common module. 
I copied over some of the necessary one to support the Preview 2 release.

Reply:
#4658